src/Stat.php

```diff
@@ -856,11 +846,7 @@ public static function showTable($itemtype, $type, $date1, $date2, $start, array
                           $data[$key2] *= $solved[$key2];
                     }
 
-                    if ($nb_solved > 0) {
-                        $timedisplay = array_sum($data) / $nb_solved;
-                    } else {
-                        $timedisplay = 0;
-                    }
+                    $timedisplay = max(array_sum($data) / $nb_solved, 0);
```

Comment:
Using `max` would not prevent a 'division by zero' fatal error.
```suggestion
                    $timedisplay = $nb_solved > 0 ? array_sum($data) / $nb_solved : 0;
```